pennylane/transforms/decompose.py

```diff
@@ -299,9 +404,22 @@ def wrapper(*inner_args):
 
 
 @partial(transform, plxpr_transform=decompose_plxpr_to_plxpr)
-def decompose(tape, gate_set=None, max_expansion=None):
+def decompose(
+    tape,
+    gate_set=None,
+    max_expansion=None,
+    fixed_decomps: dict = None,
+    alt_decomps: dict = None,
+):
     """Decomposes a quantum circuit into a user-specified gate set.
 
+    .. note::
+
+        When `qml.decomposition.enable_graph()` is present, the new experimental
+        graph-based decompositions system is enabled. This new way of doing decompositions
+        is generally more performant and allows for specifying custom decompositions.
```

Comment:
```suggestion
        is generally more resource efficient and allows for specifying custom decompositions.
```
I'll be updating this wording in my PR as well, following the suggestion from @albi3ro .